fe/fe-core/src/main/java/com/starrocks/backup/BackupJobInfo.java

```diff
@@ -147,6 +147,8 @@ public static class BackupTableInfo {
         public Long autoIncrementId;
         @SerializedName(value = "partitions")
         public Map<String, BackupPartitionInfo> partitions = Maps.newHashMap();
+        @SerializedName(value = "tableType")
+        public int tableType;
```

Comment:
Why not use Table.TableType ?

Reply:
`org.json.JSONObject` seems not support serialization and deserialization for enum type. It should be convert to INT or String to do the serialization and deserialization